packages/yourloops/components/menus/hcp-team-scope-menu.tsx

```diff
@@ -0,0 +1,217 @@
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { type FunctionComponent, useState } from 'react'
+import Box from '@mui/material/Box'
+import { makeStyles } from 'tss-react/mui'
+import { type Theme, useTheme } from '@mui/material/styles'
+import { type Team, useTeam } from '../../lib/team'
+import PersonIcon from '@mui/icons-material/Person'
+import { useSelectedTeamContext } from '../../lib/selected-team/selected-team.provider'
+import { TeamType } from '../../lib/team/models/enums/team-type.enum'
+import { useTranslation } from 'react-i18next'
+import { type TeamEditModalContentProps } from '../../pages/hcp/types'
+import { useAlert } from '../utils/snackbar'
+import GroupsOutlinedIcon from '@mui/icons-material/GroupsOutlined'
+import { useNavigate } from 'react-router-dom'
+import TeamUtils from '../../lib/team/team.util'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import IconButton from '@mui/material/IconButton'
+import Button from '@mui/material/Button'
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
+import Typography from '@mui/material/Typography'
+import MenuLayout from '../../layout/menu-layout'
+import MenuItem from '@mui/material/MenuItem'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import Divider from '@mui/material/Divider'
+import AddIcon from '@mui/icons-material/Add'
+import TeamEditDialog from '../../pages/hcp/team-edit-dialog'
+
+const classes = makeStyles()((theme: Theme) => ({
+  sectionTitle: {
+    fontWeight: 'bold',
+    paddingLeft: theme.spacing(2),
+    paddingBottom: theme.spacing(1)
+  },
+  menu: {
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(1)
+  }
+}))
+
+export const HcpTeamScopeMenu: FunctionComponent = () => {
```

Comment:
It was to be better able to differentiate it from the menu that allows a patient to go to the team settings of a team. Do you think that what is already on dblp (`TeamMenu` for the patient, and me adding `TeamScopeMenu`) would be clear enough? I was afraid it would not

Reply:
I think that it is fine without the user role in the component name :slightly_smiling_face: 
You can keep `TeamMenuSettings`, it is fine with me

Reply:
Final renamings:
- `TeamMenu` > `TeamSettingsMenu`
- `HcpTeamScopeMenu` > `TeamScopeMenu`